Sources/HaCWebsiteLib/Events/ExternalWebsiteEvent.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+struct ExternalWebsiteEvent : Event, PostCardRepresentable {
+    let title : String
+    let time : DateInterval
+    let tagLine : String
+    let color : String
+    let hypePeriod : DateInterval
+    let imageURL : String?
+    let location : Location?
+    let facebookEventID : Double?
+    var shouldShowAsUpdate: Bool {
+        get {
+            return self.hypePeriod.contains(Date())
+        }
+    }
+    var postCardRepresentation : PostCard {
+        return PostCard(
+          title: self.title,
+          category: .general,
+          description: self.tagLine,
+          backgroundColor: self.color, //TODO
+          imageURL: self.imageURL
+        )
+    }
+
+    init(called title: String, at time: DateInterval, summarised tagLine : String, colored color: String, 
```

Comment:
Often different argument labels don’t add much to the clarity of the API at the point of use. Here, the user of the API doesn’t necessarily know that `at: ` should be followed by a time. Using the parameter name `time: ` may make this clearer for the user. The same goes for (arguably) the rest of the parameters.

Reply:
You're right - I was just trying too hard and it just made things worse lol ;)